homeassistant/components/webhook/trigger.py

```diff
@@ -62,6 +83,31 @@ async def async_attach_trigger(
 ) -> CALLBACK_TYPE:
     """Trigger based on incoming webhooks."""
     webhook_id: str = config[CONF_WEBHOOK_ID]
+    local_only = config.get(CONF_LOCAL_ONLY)
+    issue_id: str | None = None
+    if local_only is None:
+        issue_id = f"trigger_missing_local_only_{webhook_id}"
+        async_create_issue(
+            hass,
+            DOMAIN,
+            issue_id,
+            breaks_in_ha_version="2023.4.0",
+            is_fixable=False,
+            learn_more_url="https://github.com/home-assistant/core/pull/66494",
+            severity=IssueSeverity.WARNING,
+            translation_key="trigger_missing_local_only",
+            translation_placeholders={
+                "webhook_id": webhook_id,
+                "automation_name": trigger_info.get("name", "[unknown]"),
```

Comment:
We could consider linking the automation directly, for an example see, the `automation` integration, which raises a repair issue for missing services (`service_not_found`).

Reply:
Nice! Thanks for that. Added in [c900929](https://github.com/home-assistant/core/pull/66494/commits/c90092917f37181fc2baf0eb1f25a4ca43becf75).